src/components/Menu/index.jsx

```diff
@@ -0,0 +1,55 @@
+import './style.css';
+
+export default function Menu() {
+  return (
+    <div className="menu">
+      <div className="dropDown">
+        <div className="dropBtn">Glaucome</div>
+        <div className="dropDownContent">
```

Comment:
Is this a button or div

Reply:
div, I don't click on to see the content, I change the classname

Reply:
To be fair, a **list** would be better as a `<ul><li>` combo anyway